cli/test/lib/tasks/download_spec.js

```diff
@@ -81,6 +81,13 @@ describe('lib/tasks/download', function () {
       snapshot('desktop url from template with version', normalize(url))
     })
 
+    it('returns custom url from template with multiple replacements', () => {
+      process.env.CYPRESS_DOWNLOAD_PATH_TEMPLATE = '${endpoint}/v${version}/${platform}/${arch}/cypress-${version}-${platform}-${arch}.zip'
```

Comment:
Should probably test multiple endpoints as well:

```suggestion
      process.env.CYPRESS_DOWNLOAD_PATH_TEMPLATE = '${endpoint}/${platform}/${arch}/cypress-${version}-${platform}-${arch}.zip?referrer=${endpoint}&version=${version}'
```